src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -103,6 +119,39 @@ export default function NftDrawer({
           }}
         >
           <p>{NFT_REWARDS_EXPLAINATION}</p>
+          <Form layout="vertical">
+            <Form.Item
+              label={
+                <TooltipLabel
```

Comment:
Can remove the tooltip component and use the `tooltip` prop on form.item instead